subprojects/ide-native/src/integTest/groovy/org/gradle/ide/visualstudio/fixtures/MSBuildExecutor.java

```diff
@@ -160,7 +161,8 @@ public ExecutionFailure fails(MSBuildAction action) {
 
     private static String fileContents(File file) {
         try {
-            return FileUtils.readFileToString(file);
+            // TODO this should not be using the default charset because it's not an input and might introduce flakiness
+            return FileUtils.readFileToString(file, Charset.defaultCharset());
```

Comment:
@gradle/native I'm not sure what to use here. Shall I create an issue for it?

Reply:
Please open an issue on gradle-native.  I suspect we can make this always be UTF8, but it might require configuring msbuild + the underlying Gradle build to be sure.